src/main/java/edu/ksu/canvas/model/assignment/Submission.java

```diff
@@ -214,4 +215,13 @@ public void setWorkflowState(String workflowState) {
         this.workflowState = workflowState;
     }
 
+    @CanvasField(postKey = "submission_history")
```

Comment:
This shouldn't have a postKey, as you can't send the submission_history when creating a submission

Reply:
True!

Reply:
Thanks.